src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/DepositCommon.ts

```diff
@@ -0,0 +1,83 @@
+import type { YearnOpportunity } from '@shapeshiftoss/investor-yearn'
+import type { DepositValues } from 'features/defi/components/Deposit/Deposit'
+import type { StakingEarnOpportunityType } from 'state/slices/opportunitiesSlice/types'
+
+type EstimatedGas = {
+  estimatedGasCrypto?: string
+}
+
+type ThorchainSaversDepositValues = DepositValues &
+  EstimatedGas & {
+    txStatus: string
+    usedGasFee: string
+  }
+
+// Redux only stores things that are serializable. Class methods are removed when put in state.
+export type SerializableOpportunity = Omit<
+  YearnOpportunity,
```

Comment:
@reallybeard

Reply:
Yes this doesn't exist yet... will be replaced when we get the stuff gomes is working on.